### PR TITLE
Add auto-deploy from circleci tag build

### PR DESCRIPTION
**Pull request by jimmidyson**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,6 +27,9 @@ ARCH ?= amd64
 BUILD_DATE := $(shell date -u)
 VERSION ?= $(shell git describe --match 'v[0-9]*' --dirty --always)
 
+OPENSHIFT_IMAGE_STREAM ?= pure-bot:deploy
+OPENSHIFT_CLIENT_IMAGE ?= openshift/origin:v1.5.0-rc.0
```
Comment (rhuss) [inline Makefile:31]: This is used where ?

**Review by rhuss — COMMENTED** [review]
Looks good except one minor thing

File: Makefile (hunk 1 above)
Comment (jimmidyson) [reply Makefile:31]: https://github.com/redhat-ipaas/pure-bot/pull/4/files#diff-b67911656ef5d18c4ae36cb6741b7965R144

File: Makefile [hunk 2]
```diff
@@ -137,6 +140,14 @@ endif
 push-name:
 	@echo "pushed: $(IMAGE):$(VERSION)"
 
+circleci-deploy: push
+	@docker run --entrypoint=/bin/sh $(OPENSHIFT_CLIENT_IMAGE) -c "                      \
+	  openshift cli login $$OPENSHIFT_SERVER --token=$$OPENSHIFT_TOKEN &&                \
+	  openshift cli project $$OPENSHIFT_PROJECT &&                                       \
+	  openshift cli tag --source=docker $(IMAGE):$(VERSION) $(OPENSHIFT_IMAGE_STREAM) && \
+	  openshift cli import-image $(OPENSHIFT_IMAGE_STREAM)
```
Comment (rhuss) [inline Makefile:148]: Missing `\`. Should I fix it ?

Comment (jimmidyson) [reply Makefile:148]: Thanks but I've sorted that in #5.

**Pull request merged**